internal/resources/cloud/resource_cloud_stack_service_account_test.go

```diff
@@ -87,24 +86,14 @@ func testAccGrafanaAuthCheckServiceAccounts(stack *gcom.FormattedApiInstance, ex
 			return fmt.Errorf("failed to get service accounts: %w", err)
 		}
 
-		var foundSAs []string
```

Comment:
This test is broken from a recent change to Grafana Cloud. Some service accounts are auto-provisioned into new stacks